content/blog/20251219_fluent-package-v5.0.9-has-been-released.md

```diff
@@ -0,0 +1,51 @@
+# fluent-package v5.0.9 has been released
+
+Hi users!
+
+We have released fluent-package [v5.0.9](https://github.com/fluent/fluent-package-builder/releases/tag/v5.0.9) on December 19, 2025.
+Fluent Package is a stable distribution package of Fluentd. (successor of td-agent v4)
+
+This is a maintenance release of v5.0.x LTS series.
+
+Fluent Package v5 LTS series is scheduled to be supported until the end of 2025.
+This release (v5.0.9) is planned to be the final version of the v5 LTS series.
```

Comment:
How about emphasizing this topic?

```html
<div markdown="span" class="alert alert-danger" role="alert">
Fluent Package v5 LTS series is scheduled to be supported until the end of 2025.
This release (v5.0.9) is planned to be the final version of the v5 LTS series.
</div>
```

In https://www.fluentd.org/blog/fluentd-v1.19.0-has-been-released article, did it about dependency conflicts.